docs/shu_bmrc.md

```diff
@@ -1,13 +1,88 @@
+## Table of Contents
 
-## Running nf-core Pipelines on the BMRC Cluster
+1. [Introduction](#introduction)
+2. [Requirements](#requirements)
+3. [Configuration Details](#configuration-details)
+4. [Usage](#usage)
+5. [Troubleshooting](#troubleshooting)
+6. [Support and Contact](#support-and-contact)
 
-A detailed guide to setting up Nextflow and nf-core and running pipelines on the BMRC Cluster is available here: [Running Nextflow with nf-core Pipelines on SHU BMRC Cluster](https://github.com/lquayle88/nfcore_on_shu_bmrc)
+---
 
-If you have any questions or issues not addressed by this guide, please contact: [l.quayle@shu.ac.uk](mailto:l.quayle@shu.ac.uk)
+## Introduction
+
+This configuration file is specifically designed for running nf-core workflows on the BMRC HPC cluster at **Sheffield Hallam University**. The configuration integrates optimal resource parameters and scheduling policies to ensure efficient job execution on the cluster, aligning with internal HPC policies and specifications.
+
+The cluster configuration:
+
+- **Location**: BMRC HPC Cluster at Sheffield Hallam University
+- **Contact**: Dr Lewis A Quayle ([l.quayle@shu.ac.uk](mailto:l.quayle@shu.ac.uk))
+- **Documentation**: [BMRC HPC Documentation](https://bmrc-hpc-documentation.readthedocs.io/en/latest/)
+
+## Requirements
+
+To use this configuration, you must have:
+
+- **Access to BMRC HPC**: Ensure your user account is enabled for HPC access at Sheffield Hallam University. The **GlobalProtect VPN** is required for remote access. For setup instructions, refer to [SHU VPN Guide](https://www.shu.ac.uk/digital-skills/programs-and-applications/virtual-private-network-vpn).
+- **Nextflow**: Version 20.10.0 or later is recommended for optimal compatibility.
+
+For a detailed guide to setting up Nextflow and running nf-core pipelines on the BMRC cluster, refer to [Running nf-core Pipelines on SHU BMRC Cluster](https://bmrc-hpc-documentation.readthedocs.io/en/latest/nfcore/index.html).
+
+## Configuration Details
+
+The configuration has been tailored for the BMRC HPC, providing preset values for CPUs, memory, and scheduling to align with HPC policies.
+
+### Core Configuration
+
+- **Cluster Scheduler**: `slurm`
+- **Max Retries**: 2 (automatically reattempts failed jobs)
+- **Queue Size**: 50 jobs
+- **Submit Rate Limit**: 1 job per second
+
+### Resource Allocation
+
+Each nf-core workflows will automatically receive the following default resource maxima:
+
+| Resource | Setting   |
+| -------- | --------- |
+| CPUs     | 64        |
+| Memory   | 1007 GB   |
+| Time     | 999 hours |
+
+### Container Support
+
+The configuration supports Apptainer for containerised workflows, with automatic mounting enabled, allowing seamless access to necessary filesystems within containers.
+
+### Cleanup
+
+Intermediate files from successful runs will be automatically deleted to free up storage.
```

Comment:
You can _maybe_ mention if you need to debug a run without cleaning up  by including the `debug` profile, if you trust your users :grimacing:

Reply:
Most of them are new HPC users, so I'll maybe leave this as is for the time being